cluster/saltbase/salt/kube-apiserver/initd

```diff
@@ -41,6 +41,9 @@ do_start()
         # Raise the file descriptor limit - we expect to open a lot of sockets!
         ulimit -n 65536
 
+        # Make it possible to bind to low numbered ports.
+        /sbin/setcap 'cap_net_bind_service=+ep' $DAEMON
```

Comment:
Now the the apiserver is running in a container, this has cleaned up quite nicely.